core/mm/core_mmu.c

```diff
@@ -1016,6 +1060,11 @@ static size_t collect_mem_ranges(struct tee_mmap_region *memory_map,
 
 #undef ADD_PHYS_MEM
 
+#if defined(CFG_CORE_SEL2_SPMC) && defined(CFG_CORE_PHYS_RELOCATABLE)
```

Comment:
`if (IS_ENABLED(CFG_CORE_SEL2_SPMC))`
I think this should always be supported with an S-EL2 SPMC, regardless of `CFG_CORE_PHYS_RELOCATABLE`.

Reply:
The manifest_dtb is obtained from boot_save_boot_info. Only defined(CFG_CORE_SEL2_SPMC) && defined(CFG_CORE_PHYS_RELOCATABLE) executes boot_save_boot_info. Can I try to validate boot_save_boot_info related to CFG_CORE_PHYS_RELOCATABLE with this macro, and the rest of the code only with CFG_CORE_SEL2_SPMC?
How about：
```
...
void __weak boot_save_boot_info(void *boot_info)
{
	void *fdt = NULL;
	paddr_t base = 0;
	size_t size = 0;

	fdt = get_fdt_from_boot_info(boot_info);
#if defined(CFG_CORE_PHYS_RELOCATABLE)
	get_sec_mem_from_manifest(fdt, &base, &size);
	core_mmu_set_secure_memory(base, size);
#endif
}
#endif /*CFG_CORE_SEL2_SPMC*/
```